Conceptpower+Spring/pom.xml

```diff
@@ -23,8 +23,9 @@
 		<email.server.host>http://example.net/</email.server.host>
 		<admin.username>admin</admin.username>
 		<admin.password>admin</admin.password>
-		<wordnet.path>/usr/local/WordNet-3.0</wordnet.path>
+		<wordnet.path>C:/Program Files (x86)/WordNet/2.1</wordnet.path>
```

Comment:
@jdamerow  I was working on windows machine sometime back. Changed back to the correct location